dbus/src/nonblock/generated_org_freedesktop_dbus.rs

```diff
@@ -1,6 +1,6 @@
-// This code was autogenerated with `dbus-codegen-rust -d org.freedesktop.DBus -p /org/freedesktop/DBus -m none -c nonblock -g -i org.freedesktop. --dbuscrate crate --filter DBus -o dbus/src/nonblock/generated_org_freedesktop_dbus.rs`, see https://github.com/diwic/dbus-rs
-#![allow(missing_docs)]
+// This code was autogenerated with `dbus-codegen-rust -d org.freedesktop.DBus -p /org/freedesktop/DBus -m none -c nonblock -g -i org.freedesktop. --dbuscrate crate --interfaces DBus -o dbus/src/nonblock/generated_org_freedesktop_dbus.rs`, see https://github.com/diwic/dbus-rs
 use crate as dbus;
+#[allow(unused_imports)]
 use crate::arg;
```

Comment:
Maybe `#[allow(unused_imports)]` should be a part of what we generate?

Reply:
It is! :D